### ISSUE TITLE
Log spammed with HTTP 404 for IVT with 0.19.3

### ISSUE THREAD

**Issue by slovdahl** [bosch-thermostat/home-assistant-bosch-custom-component#214]
**Describe the bug**

The log is spammed with 6 warnings like `WARNING (MainThread) [bosch_thermostat_client.helper] Can't update data for Current room setpoint. Trying uri: /heatingCircuits/hc1/roomtemperature. Error message: URI /heatingCircuits/hc1/roomtemperature doesn not exist: 404, message='Not Found', url=URL('http://x.x.x.x/heatingCircuits/hc1/roomtemperature')` once a minute since upgrading to 0.19.3.

Downgrading back to 0.18.3 fixes the logging.

**To Reproduce**

Start HA and observe the log.

**Expected behavior**

No warnings are logged.

**Screenshots**

With debug logging from my dev instance:

```
2022-10-11 18:29:19.083 DEBUG (MainThread) [custom_components.bosch] Updating component sensor sensor.hc1_summer_winter_threshold by 140596499242432
2022-10-11 18:29:19.084 DEBUG (MainThread) [bosch_thermostat_client.connectors.http] Sending GET request to /heatingCircuits/hc1/roomtemperature
2022-10-11 18:29:19.150 WARNING (MainThread) [bosch_thermostat_client.helper] Can't update data for Summer Winter Threshold. Trying uri: /heatingCircuits/hc1/roomtemperature. Error message: URI /heatingCircuits/hc1/roomtemperature doesn not exist: 404, message='Not Found', url=URL('http://x.x.x.x/heatingCircuits/hc1/roomtemperature')
2022-10-11 18:29:19.152 DEBUG (MainThread) [custom_components.bosch] Updating component sensor sensor.hc1_pump_modulation by 140596499242432
2022-10-11 18:29:19.152 DEBUG (MainThread) [bosch_thermostat_client.connectors.http] Sending GET request to /heatingCircuits/hc1/roomtemperature
2022-10-11 18:29:19.230 WARNING (MainThread) [bosch_thermostat_client.helper] Can't update data for Pump Modulation. Trying uri: /heatingCircuits/hc1/roomtemperature. Error message: URI /heatingCircuits/hc1/roomtemperature doesn not exist: 404, message='Not Found', url=URL('http://x.x.x.x/heatingCircuits/hc1/roomtemperature')
2022-10-11 18:29:19.231 DEBUG (MainThread) [custom_components.bosch] Updating component sensor sensor.hc1_summer_winter_switchmode by 140596499242432
2022-10-11 18:29:19.232 DEBUG (MainThread) [bosch_thermostat_client.connectors.http] Sending GET request to /heatingCircuits/hc1/roomtemperature
2022-10-11 18:29:19.321 WARNING (MainThread) [bosch_thermostat_client.helper] Can't update data for Summer Winter Switchmode. Trying uri: /heatingCircuits/hc1/roomtemperature. Error message: URI /heatingCircuits/hc1/roomtemperature doesn not exist: 404, message='Not Found', url=URL('http://x.x.x.x/heatingCircuits/hc1/roomtemperature')
2022-10-11 18:29:19.322 DEBUG (MainThread) [custom_components.bosch] Updating component sensor sensor.hc1_current_summer_winter_mode by 140596499242432
2022-10-11 18:29:19.322 DEBUG (MainThread) [bosch_thermostat_client.connectors.http] Sending GET request to /heatingCircuits/hc1/roomtemperature
2022-10-11 18:29:19.401 WARNING (MainThread) [bosch_thermostat_client.helper] Can't update data for Current Summer Winter Mode. Trying uri: /heatingCircuits/hc1/roomtemperature. Error message: URI /heatingCircuits/hc1/roomtemperature doesn not exist: 404, message='Not Found', url=URL('http://x.x.x.x/heatingCircuits/hc1/roomtemperature')
2022-10-11 18:29:19.402 DEBUG (MainThread) [custom_components.bosch] Updating component sensor sensor.hc1_actualsupplytemperature by 140596499242432
2022-10-11 18:29:19.402 DEBUG (MainThread) [bosch_thermostat_client.connectors.http] Sending GET request to /heatingCircuits/hc1/roomtemperature
2022-10-11 18:29:19.490 WARNING (MainThread) [bosch_thermostat_client.helper] Can't update data for actualSupplyTemperature. Trying uri: /heatingCircuits/hc1/roomtemperature. Error message: URI /heatingCircuits/hc1/roomtemperature doesn not exist: 404, message='Not Found', url=URL('http://x.x.x.x/heatingCircuits/hc1/roomtemperature')
2022-10-11 18:29:19.492 DEBUG (MainThread) [custom_components.bosch] Updating component sensor sensor.hc1_current_room_setpoint by 140596499242432
2022-10-11 18:29:19.492 DEBUG (MainThread) [bosch_thermostat_client.connectors.http] Sending GET request to /heatingCircuits/hc1/roomtemperature
2022-10-11 18:29:19.571 WARNING (MainThread) [bosch_thermostat_client.helper] Can't update data for Current room setpoint. Trying uri: /heatingCircuits/hc1/roomtemperature. Error message: URI /heatingCircuits/hc1/roomtemperature doesn not exist: 404, message='Not Found', url=URL('http://x.x.x.x/heatingCircuits/hc1/roomtemperature')
2022-10-11 18:29:19.573 DEBUG (MainThread) [custom_components.bosch] Bosch sensor entitites updated.
```

**Version**
  - HA version? 2022.10.2
  - HA Bosch component version? 0.19.3
  - Python Bosch HTTP client version? 0.19.3

** Debug SCAN **
** IMPORTANT **

[bosch_scan.txt](https://github.com/bosch-thermostat/home-assistant-bosch-custom-component/files/9759286/bosch_scan.txt)

**Additional context**

**Comment by pszafer** [comment]
I need bosch scan, so I can try to reproduce bug

**Comment by rgerbranda** [comment]
I stopped spamming the logs by disabling all sensor.hc1_*

<img width="578" alt="afbeelding" src="https://user-images.githubusercontent.com/11868439/195180508-4d219029-0def-4e00-bed1-e5b987300842.png">

**Comment by larsdunemark** [comment]
Have the same issue with my IVT PremiumLine EQ.
Attaching my scan file.
[bosch_scan.txt](https://github.com/bosch-thermostat/home-assistant-bosch-custom-component/files/9759179/bosch_scan.txt)

**Comment by slovdahl** [comment]
Sorry, here's mine: [bosch_scan.txt](https://github.com/bosch-thermostat/home-assistant-bosch-custom-component/files/9759286/bosch_scan.txt)

**Comment by PeterHolgersson** [comment]
I have the same problem.
Let me know if you need my log aswell.

**Comment by pszafer** [comment]
Fixed in 0.19.4
